src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AttributeNamedParameterCompletionProviderTests.cs

```diff
@@ -85,7 +80,8 @@ public class TestAttribute : Attribute
     public ConsoleColor Color { get; set; }
 }";
 
-            await VerifyItemExistsAsync(markup, "Color", displayTextSuffix: " =");
+            using var workspaceFixture = CreateWorkspaceFixture();
```

Comment:
> So wasn't the point of the fixture being shared between tests so we would keep our test runtimes down?

That was the goal a _long_ time ago. The cross-test cached data itself was eliminated back in 12a789658c012627c5b91de699efdabd9cef45d7 (#25681). I avoided this refactoring at the time because it was so tedious, but it became necessary now that the test cleanup relies more heavily on tests to dispose workspaces as part of cancelling work for test performance.